lib/models/asyncapi.js

```diff
@@ -208,18 +209,18 @@ class AsyncAPIDocument extends Base {
   /**
    * Callback used when crawling a schema.
    * @callback TraverseSchemas
```

Comment:
I am not sure if this will work. Try the following:
```suggestion
   * @callback TraverseSchemas
   * @alias module:@asyncapi/parser#TraverseSchemas
```

Reply:
Forgot that we wraps everything into the module alias... Which is why just requiring the Schema does not make sense🙂

Reply:
@jonaslagoni Same error. :(

Reply:
Try re-generate the types as well 🙂

Reply:
Or where there no changes? 🤔

Reply:
> Try re-generate the types as well

already done. Schema is still undefined in types.d.ts.

Reply:
Hmm...